demos/python_demos/speech_recognition_demo/ctcdecode-numpy/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ endforeach()
 # SWIG-generated code causes some warnings; disable them.
 if(COMPILER_IS_GCC_LIKE)
     target_compile_options(${target_name} PRIVATE -Wno-narrowing)
+elseif(MSVC)
+    target_compile_options(${target_name} PRIVATE /W0)
```

Comment:
Surely it's not necessary to disable _all_ warnings? I don't think this library is that hopeless. :-)

Reply:
Yes, but now I think that we shouldn't introduce differences with the original code without a strong reason so it would be easier to compare them if the original one introduces something what we want.

Reply:
What do you mean by the original code?

Reply:
The one these files were copied from.

Reply:
I should note that not all of these files were copied; several of them are original to us, including `word_prefix_set.cpp`, where that 4834 warning occurs.

Reply:
Fixed